.github/workflows/ci.yaml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.7"
+          
+      - run: python3 -m pip install --upgrade pip setuptools wheel
+      - run: python3 -m pip install nextstrain-cli
+      - run: nextstrain version
+      - run: nextstrain check-setup
+      - run: nextstrain update
+      - name: Copy example data
+        run: |
+          mkdir -p data/
+          cp -v example_data/* data/
+
+      - run: nextstrain build --docker . auspice/flu_avian_h5n1_ha.json
```

Comment:
Looks like it to me. I'm inclined to first parameterize the shared/reusable workflow and then use it here instead of making this change now and then another update here later too.